src/controllers/AdminController.php

```diff
@@ -3341,10 +3341,7 @@ class={$highlighted_color}
 
           $level = $results->get('level');
           invariant($level !== null, 'Level should not be null');
-          invariant(
-            $level instanceof Level,
-            'level should be of type Level',
-          );
+          invariant($level instanceof Level, 'level should be of type Level');
```

Comment:
This is a minor change.  Not sure why the `format.sh` script is treating this differently now.